src/context-menu/CanvasContextMenu.tsx

```diff
@@ -30,6 +31,21 @@ export class CanvasContextMenu extends React.Component<CanvasContextMenuProps> {
             });
         };
 
+        const handleAttachNode = async () => {
+            this.props.app.commands.execute(commandIDs.attachNode);
```

Comment:
Is there a reason we don't await the first execution too?

Reply:
Good question. The attach / detach is quite light, but to be safe we should await it.
Updated.